### ISSUE TITLE
Blazor WebAssembly SignalR logging not working

### ISSUE THREAD

**Issue by WayneHiller** [dotnet/aspnetcore#31516]
.Net 5.0.4

I Have a Blazor WebAssembly App that is Using SignalR. The SignalR code is working but I cannot seem to get any logging to work. Nothing shows in the Browser Console.

Index.razor.cs

```
        [Inject]
        public NavigationManager NavMgr { get; set; }

        [Inject]
        public ILoggerProvider LogProvider { get; set; }

...

        protected override async Task OnInitializedAsync()
        {
            await base.OnInitializedAsync();

            var connection = new HubConnectionBuilder()
                .WithUrl(NavMgr.ToAbsoluteUri("/exporthub"))
                .WithAutomaticReconnect()
                .ConfigureLogging(logging => {
                    logging.AddProvider(LogProvider);
                })
                .Build();

            await connection.StartAsync();
        }

```

Is there something else I need to do?

**Comment by BrennanConroy** [comment]
I don't know how console logging and blazor webassembly work together here:
```
[Inject]
public ILoggerProvider LogProvider { get; set; }
```

You might want to consider setting the minimum log level here too.
```
.ConfigureLogging(logging => {
    logging.AddProvider(LogProvider);
})
```

**Comment by WayneHiller** [comment]
@BrennanConroy I followed the direction in here https://docs.microsoft.com/en-us/aspnet/core/blazor/fundamentals/logging?view=aspnetcore-5.0&pivots=webassembly#signalr-net-client-logging

As I understand it the
```
[Inject]
public ILoggerProvider LogProvider { get; set; }
```

Will inject a WebAssemblyConsoleLogger which will log to the browser console.
I did try setting the minimum level in the ConfigureLogging SignalR call, no help. I get logging from the app if I set the builder.Logging.SetMinimumLevel(LogLevel.Trace); in the program.cs file but no SignalR logging.
